cask-repair

```diff
@@ -561,20 +566,19 @@ for cask in "${@}"; do
 
   git commit "${cask_file}" --message "${commit_message}" --quiet
   git push --force "${cask_repair_remote_name}" "${cask_branch}" --quiet 2> "${submission_error_log}"
-  gitpush_result="${?}"
-
-  # Fix common push errors
-  if [[ "${gitpush_result}" -ne 0 ]] && grep --quiet 'shallow update not allowed' "${submission_error_log}"; then
-    echo 'Push failed due to shallow repo. Unshallowing…'
-    HOMEBREW_NO_AUTO_UPDATE=1 brew tap --full "caskroom/$(current_tap)"
-    git push --force "${cask_repair_remote_name}" "${cask_branch}" --quiet 2> "${submission_error_log}"
-    gitpush_result="${?}"
-  fi
 
-  if [[ "${gitpush_result}" -ne 0 ]]; then
-    warning_message 'There were errors while pushing:'
-    echo "$(< "${submission_error_log}")"
-    abort 'Please fix the errors and try again. If the issue persists, open a bug report on the repo for this script (https://github.com/vitorgalvao/tiny-scripts).'
+  if [[ "${?}" -ne 0 ]]; then
+    # Fix common push errors
+    if grep --quiet 'shallow update not allowed' "${submission_error_log}"; then
+      echo 'Push failed due to shallow repo. Unshallowing…'
+      HOMEBREW_NO_AUTO_UPDATE=1 brew tap --full "caskroom/$(current_tap)"
+      git push --force "${cask_repair_remote_name}" "${cask_branch}" --quiet 2> "${submission_error_log}"
+      if [[ "${?}" -ne 0 ]]; then
```

Comment:
Add an empty line before it and replace the block with `[[ "${?}" -ne 0 ]] && push_failure_message "$(< "${submission_error_log}")"`.

After that, I think we’re good to go!

Reply:
ok